docs/pl/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Strona główna
+meta:
+  - property: og:title
+    content: Strona główna | Ethereum
+lang: pl-PL
```

Comment:
Same with this `lang` property, you can change to `pl` for consitency (you can likely find & replace all `pl-PL` with `pl`.